packages/next/build/webpack-config.ts

```diff
@@ -356,7 +356,16 @@ export default async function getBaseWebpackConfig(
       'next/router': 'next/dist/client/router.js',
       'next/config': 'next/dist/next-server/lib/runtime-config.js',
       'next/dynamic': 'next/dist/next-server/lib/dynamic.js',
-      next: NEXT_PROJECT_ROOT,
+      ...(isServer
+        ? {}
+        : {
+            stream: 'stream-browserify',
+            path: 'path-browserify',
+            crypto: 'crypto-browserify',
+            buffer: 'buffer',
+            vm: 'vm-browserify',
```

Comment:
@timneutkens These should go through`require.resolve`

Reply:
Great point, forgot those don't resolve automatically. Opened #16033 with you as co-author.